### ISSUE TITLE
The Counter <button> in default template has color contrast issues

### ISSUE THREAD

**Issue by bamadesigner** [sveltejs/kit#961]
**Describe the bug**
The Counter <button> in default template has color contrast issues.

The button has insufficient color contrast of 3.08 (foreground color: #ff3e00, background color: #ffece6, font size: 12.0pt (16px), font weight: normal). The expected contrast ratio is 4.5:1.

**To Reproduce**
Download the app and run the development environment. The button exists in the index route.

**Expected behavior**
The contrast between the button background color and the button text color should have a ratio of 4.5:1. Right now it is 3.08.

**Severity**
This is an accessibility violation and will affect some users who have low vision, color blindness, etc.

**Additional context**
As this is example code that users might replicate, we should make sure it is accessible for all.

<img width="233" alt="Screen Shot of button with color contrast issues" src="https://user-images.githubusercontent.com/1513046/114282368-89aefc00-9a11-11eb-80d6-32aca9376c5e.png">

**Comment by GreenRobot777** [comment]
`<offtop>`
@bamadesigner Can you send an link to some tools or projects that detect thouse a11y issues (for buttons, for example) automatically?
`</offtop>`

**Comment by GreenRobot777** [comment]
@bamadesigner [This change](https://github.com/sveltejs/kit/pull/1014/files#diff-f6855cf2abc6352ad95a74c0269d1931d1c6ad315480d3dc4ea3319e68319926R7-R14) is fix this issue?

**Comment by Rich-Harris** [comment]
The demo app has been replaced, so I'll close this (but any a11y-related issues pertaining to the new app are more than welcome) — thank you

https://netlify.demo.svelte.dev/
